libs/cardano-ledger-test/src/Test/Cardano/Ledger/STS.hs

```diff
@@ -286,64 +312,111 @@ genDelegCert pp ds =
 -- Properties
 ------------------------------------------------------------------------
 
+-- Setup ------------------------------------------------------------------
+
+data PropEnv era = PropEnv
+  { net :: Network
+  , pParams :: PParams era
+  , epochNo :: EpochNo
+  , slotNo :: SlotNo
+  }
+
+type GenShrink a = (Gen a, a -> [a])
+
+stsProperty ::
+  forall r era env st sig.
+  ( Environment (EraRule r era) ~ env
+  , State (EraRule r era) ~ st
+  , Signal (EraRule r era) ~ sig
+  , STS (EraRule r era)
+  , era ~ ConwayEra StandardCrypto
```

Comment:
Yes, it's temporary but fixing it might touch a bunch of other stuff - or it's a leftover from the rewrite that never got removed. I'll take a stab at fixing it today.